deploy/scripts/teams/team.csv

```diff
@@ -0,0 +1,2 @@
+Rene Hungry;rhungry@mirantis.com;18234243156;ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQ....nkIT5ZgtlML rhungry@hungry-man.local
```

Comment:
It would be good to don't add this file or just something dummy (fake names, etc.)